botocore/utils.py

```diff
@@ -130,6 +130,22 @@
 }
 
 
+class _RetriesExceededError(Exception):
```

Comment:
Why is this needed?

Reply:
Its not. I think it is the result of fixing merge conflicts where the `_RetriesExceededError` got moved in the `utils` module and now I have duplicates of it. I'll fix it.